### ISSUE TITLE
Please add info on standard text attribute names, values and default values

### ISSUE THREAD

**Issue by aleventhal** [LinuxA11y/IAccessible2#7]
The only place I could find where this is currently documented is here:
https://wiki.linuxfoundation.org/accessibility/iaccessible2/textattributes

**Comment by aleventhal** [comment]
@jcsteh friendly ping :)

**Comment by aleventhal** [comment]
Ok. How about we at least link to it in the readme for this project. It took me ages to find it, I'll say that! Joanie didn't even know about it, and I had to dig for quite a while.

**Comment by asurkov** [comment]
> Ok. How about we at least link to it in the readme for this project. It took me ages to find it, I'll say that! Joanie didn't even know about it, and I had to dig for quite a while.

Agreed. I recall finding myself in same situation. It's next to impossible to find that doc. And it's good to have readme file.

**Comment by asurkov** [comment]
filed PR to update README to include the links https://github.com/LinuxA11y/IAccessible2/pull/9
